### DISCUSSION TITLE
Consulta Categoría de entidad

### DISCUSSION THREAD

**Discussion by picardovidal** [dds-jv/foro#152] [TPA - 2020] [answered]
En la primer entrega del TP las entidades se categorizaron según estos puntos:
13. Las entidades jurídicas serán categorizadas en Empresas y OSC (Organizaciones del sector
social).
14. En el caso de empresas, estas se clasifican en Micro, Pequeña, Mediana Tramo 1, Mediana Tramo 2. Dicha clasificación responderá a los criterios estipulados por la AFIP

Las nuevas categorías deben reemplazar a estas? o son otra categoría a manejar en paralelo?

**Comment by flbulgarelli** [comment, marked as answer]
¡Hola!

Son conceptos independientes y no relacionados, que se llamen categoría es pura casualidad.

De hecho incluso en el enunciado de la primera entrega usa de forma intercambiable _clasificar_ y _categorizar_, como también incluso podría haber dicho _tipificar_ o _agrupar_, y no necesariamente emana del enunciado el concepto de categorías.

Acá lo te que te va a convenir hacer es elegir un nombre nuevo para la abstracción de esta entrega, renombrar las abstracción original o ambas cosas. Queda a tu criterio, solo asegurate de elegir nombres que les resulten claros en su equipo, y de usarlos de forma consistente.

¡Saludos!

**Comment by picardovidal** [comment]
Gracias Franco!
